include/ccf/service/tables/jsengine.h

```diff
@@ -12,11 +12,13 @@ namespace ccf
     size_t max_heap_bytes;
     /// @brief stack size for QuickJS runtime
     size_t max_stack_bytes;
+    /// @brief max execution time for QuickJS
+    uint64_t max_execution_time;
```

Comment:
As for the other values, we should include the unit in the name.

Reply:
Done